Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -523,6 +528,9 @@ class BrowserViewController: UIViewController {
 
     override func viewWillDisappear(animated: Bool) {
         screenshotHelper.viewIsVisible = false
+        
+        log.debug("BVC stopping handoff.")
+        HandoffManager.sharedInstance.stop()
```

Comment:
@sleroux

I considered this operation.

Steps
* 1. View a web page. (not blank tab)
* 2. Press the tab switch button. (on the right top)
  - 2-a. Press the config button.
  - 2-b. Press the Incognito Mode button.


I think handoff should stop in the following situation. 
This code is intended to stop this.

2.
User may not understand which page in the tab list was sent
2-a.
Sending a link while showing the config page isn't intended.
2-b.
Users may misunderstand that link of incognito page was sent.


---
2
![2](https://cloud.githubusercontent.com/assets/1459450/12699902/39e1d116-c811-11e5-9e2e-b72f6ecfaf48.png)


2a
![2a](https://cloud.githubusercontent.com/assets/1459450/12699906/4590cd14-c811-11e5-9378-480e2e637c3d.png)

2b
![2b](https://cloud.githubusercontent.com/assets/1459450/12699907/45917188-c811-11e5-8608-dd7cedab79f9.png)